RunStuff Template.m

```diff
@@ -0,0 +1,165 @@
+(* ::Package:: *)
```

Comment:
What is the difference between the template and the regular file?

Reply:
Only keep runstufftemplate (this contains tests)